dm/worker/server.go

```diff
@@ -288,7 +288,10 @@ func (s *Server) StartSubTask(ctx context.Context, req *pb.StartSubTaskRequest)
 
 	if resp.Result {
 		op1 := clientv3.OpPut(common.UpstreamSubTaskKeyAdapter.Encode(cfg.SourceID, cfg.Name), req.Task)
-		resp.Msg = s.retryWriteEctd(op1)
+		err = s.retryWriteEctd(op1)
+		if err != nil {
+			return nil, err
```

Comment:
Do not return a `resp` with `err` here? The same with L328 and L778.

Reply:
If gRPC service returns an `err` here, dm-master will not check about the resp.

Reply:
So why don't we unify the return values ​​here?

Reply:
`return resp, err`?

Reply:
Can not `return resp, nil`?